recipes/arrow/all/conanfile.py

```diff
@@ -637,7 +637,11 @@ def package_info(self):
         if self.options.with_mimalloc:
             self.cpp_info.components["libarrow"].requires.append("mimalloc::mimalloc")
         if self._with_re2():
-            self.cpp_info.components["libgandiva"].requires.append("re2::re2")
+            if self.options.gandiva:
```

Comment:
@kou could you please confirm - does this change make sense?

Reply:
Yes. It makes sence.

Reply:
Thanks you taking the time to answer!